tests/gpts/github/test-config/github_positive_suite.ts

```diff
@@ -195,7 +195,20 @@ export const gitHubBasicGoldenPathTemplateTests = (gptTemplate: string) => {
                 expect(finished).toBe(true)
             }
         }, 900000)
-
+  
+         /**
+         * Check if the pipelinerun yaml has the rh-syft image path mentioned
+         */
+         it(`Check ${gptTemplate} pipelinerun yaml has the rh-syft image path`, async ()=> {
+            const pipelineRun = await kubeClient.getPipelineRunByRepository(repositoryName, 'push')
+            if (pipelineRun && pipelineRun.metadata && pipelineRun.metadata.name) {
+                const doc = await kubeClient.pipelinerunfromName(pipelineRun.metadata.name,developmentNamespace)
+                const image_name = doc.spec.pipelineSpec.tasks[2].taskSpec.steps[1].image
+            console.log(image_name)
+            expect(image_name).not.toBe(undefined)
```

Comment:
You are checking, if the image name is defined, not it it is rh-syft-* as the description/comment suggest.

Reply:
Added the search for the string "registry.redhat.io/rh-syft-tech-preview/syft-rhel9"